src/Tools/CommandLine.hs

```diff
@@ -0,0 +1,99 @@
+module Tools.CommandLine where
+
+import           System.IO                      ( hPutStrLn
+                                                , stderr
+                                                )
+import           System.Exit                    ( exitFailure )
+import           Data.Version
+import           Options.Applicative
+import           Data.Semigroup                 ( (<>) )
+import qualified Data.Text                     as T
+
+import           Constants.Flags
+import           Paths_madlib                  as Lib
+
+hashBar = "################################################"
+h1 = " ____    __  ____   _____   ____    ____  ______"
+h2 = "|    \\  /  ||    \\  |    \\ |    |  |    ||      )"
+h3 = "|     \\/   ||     \\ |     \\|    |_ |    ||     <"
+h4 = "|__/\\__/|__||__|\\__\\|_____/|______||____||______)"
+headerString = h1 ++ "\n" ++ h2 ++ "\n" ++ h3 ++ "\n" ++ h4
+
+-- -- parseCommand :: Parser Command
```

Comment:
Before we merge the PR I'd just extract what you want to keep in another branch and remove all comments from here.

Reply:
Oh definitely, this file has had 4 lives already